.prettierrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "bracketSpacing": false,
+  "printWidth": 80,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "useTabs": false,
+  "arrowParens": "always"
+}
```

Comment:
Any reason why we don't just use the defaults from prettier? Also is this consistent with other chai projects?

Reply:
this is the same as the one in other chai projects